### PR TITLE
chore: add few hindi messages translations

### PR DESCRIPTION
**Pull request by maze-runnar**
related #5217 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by vsasvipul0605 — APPROVED** [review]

File: translations/hi.po [hunk 1]
```diff
@@ -6825,103 +6830,146 @@ msgstr ""
...
+
+#~ msgid "on our servers"
+#~ msgstr "on our servers"
+
+#~ msgid "Street Address Line 1"
+#~ msgstr "Street Address Line 1"
+
+#~ msgid "Street Address Line 2"
+#~ msgstr "Street Address Line 2"
+
```
Comment (iamareebjamal) [inline translations/hi.po:6975]:
```suggestion
```

File: translations/hi.po [hunk 2]
```diff
@@ -1576,177 +1576,182 @@ msgstr ""
...
+msgstr "एक अप्रत्याशित त्रुटि हुई"
 
 #: app/controllers/public/cfs/edit-session.js:12:28
 msgid "Session details have been saved"
-msgstr ""
+msgstr "सत्र विवरण सहेजे गए हैं"
+
+#: app/controllers/public/index.js:121:28
+msgid "Order details saved. Please fill further details within {{time}} minutes."
+msgstr "आदेश विवरण सहेजे गए। कृपया {{समय}} मिनटों के भीतर और विवरण भरें।"
```
Comment (iamareebjamal) [inline translations/hi.po:1750]: This is a placeholder

```suggestion
msgstr "आदेश विवरण सहेजे गए। कृपया {{time}} मिनटों के भीतर और विवरण भरें।"
```

File: translations/hi.po [hunk 3]
```diff
@@ -6825,103 +6830,103 @@ msgstr ""
...
+msgstr "द्वारा आयोजित"
 
 #: app/templates/public/index.hbs:74:10
 msgid "Online Event"
 msgstr "ऑनलाइन कार्यक्रम"
 
 #: app/templates/public/index.hbs:87:10
 msgid "Location to be announced"
-msgstr ""
+msgstr "घोषित किए जाने का स्थान"
```
Comment (iamareebjamal) [inline translations/hi.po:6893]: Wrong

File: translations/hi.po [hunk 4]
```diff
@@ -1576,177 +1576,182 @@ msgstr ""
...
 #: app/controllers/events/view/tickets/access-codes/list.js:72:28
 msgid "Access Code has been updated successfully."
-msgstr ""
+msgstr "एक्सेस कोड सफलतापूर्वक अपडेट किया गया है।"
 
 #: app/controllers/events/view/tickets/add-order.js:23:28
 #: app/controllers/public/index.js:121:28
 msgid "Order details saved. Please fill further details within {{time}} minutes."
-msgstr ""
+msgstr "आदेश विवरण सहेजे गए। कृपया 10 मिनट के भीतर और विवरण भरें।"
```
Comment (iamareebjamal) [inline translations/hi.po:1645]: Placeholder missing and there is a duplicate translation below

**Review by akashgp09 — COMMENTED** [review]
LGTM 👍

File: translations/hi.po [hunk 5]
```diff
@@ -454,574 +454,574 @@ msgstr "कृपया एक टैग लाइन दर्ज करें"
...
 #: app/components/forms/orders/order-form.js:439:23
 msgid "Please specify your choice of payment method"
-msgstr ""
+msgstr "कृपया अपनी भुगतान विधि का चयन करें"
 
 #: app/components/forms/orders/order-form.js:479:23
 #: app/components/forms/session-speaker-form.js:390:23
 msgid "Please enter {{field}}"
-msgstr ""
+msgstr "कृपया {{फ़ील्ड}} डालें"
```
Comment (iamareebjamal) [inline translations/hi.po:742]:
```suggestion
msgstr "कृपया {{field}} डालें"
```

File: translations/hi.po [hunk 6]
```diff
@@ -1188,312 +1188,312 @@ msgstr "रद्द करना"
...
 
 #: app/controllers/admin/content/events.js:51:26
 msgid "An unexpected error has occurred. Event Type was not deleted."
-msgstr ""
+msgstr "कोई अनपेक्षित त्रुटि हुई है। ईवेंट प्रकार हटाया नहीं गया था।"
 
 #: app/controllers/admin/content/events.js:68:28
 msgid "{{item}} has been added successfully."
-msgstr ""
+msgstr "{{आइटम}} सफलतापूर्वक जोड़ा गया है।"
```
Comment (iamareebjamal) [inline translations/hi.po:1400]:
```suggestion
msgstr "{{item}} सफलतापूर्वक जोड़ा गया है।"
```

File: translations/hi.po [hunk 7]
```diff
@@ -1188,312 +1188,312 @@ msgstr "रद्द करना"
...
 
 #: app/controllers/admin/content/events.js:77:77
 msgid "{{item}} not saved."
-msgstr ""
+msgstr "{{आइटम}} सहेजा नहीं गया।"
```
Comment (iamareebjamal) [inline translations/hi.po:1404]:
```suggestion
msgstr "{{item}} सहेजा नहीं गया।"
```

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**